README.md

```diff
@@ -124,9 +124,11 @@ Deploy Station with [Docker](https://www.docker.com/). Please replace
 `FIL_WALLET_ADDRESS`.
 
 ```bash
-$ docker run \
-	-e FIL_WALLET_ADDRESS=f1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za \
-	-d ghcr.io/filecoin-station/core
+# Run container
+$ docker run --name station -d -e FIL_WALLET_ADDRESS=f1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za ghcr.io/filecoin-station/core
```

Comment:
Nitpick: I prefer the original version that splits the long line into multiple logical segments.